design-patterns/lookup-single-record-by-name/README.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 8
+name: Filter Records by Name
+description: Users can query multiple  using named slots and ids
```

Comment:
Let's maintain the same tense as other design patterns.

Example: "Take an action on a business object, referencing the ID of the object."